### PR TITLE
Added example for map_union and fixed link 

### PR DESCRIPTION
**Pull request by ckodonnell**
@mosabua @Ordinant @rosewms @m57lyra

### CODE REVIEW

File: docs/src/main/sphinx/functions/aggregate.rst [hunk 1]
```diff
@@ -213,6 +213,30 @@ Map aggregate functions
    Returns the union of all the input maps. If a key is found in multiple
    input maps, that key's value in the resulting map comes from an arbitrary input map.
 
+   For example, take the following histogram function that creates multiple maps from the Iris dataset::
+
+        SELECT histogram(floor(petal_length_cm)) x
```
Comment (mosabua) [inline docs/src/main/sphinx/functions/aggregate.rst:218]: use a better column name than x

File: docs/src/main/sphinx/functions/aggregate.rst [hunk 2]
```diff
@@ -213,6 +213,30 @@ Map aggregate functions
...
+        FROM memory.default.iris
+        GROUP BY species;
+                    x
```
Comment (mosabua) [inline docs/src/main/sphinx/functions/aggregate.rst:221]: add empty line to make it more readable

**Review by mosabua — CHANGES_REQUESTED** [review]
Great start!

**Comment by mosabua** [comment]
Fixes https://github.com/trinodb/trino/issues/7663

**Comment by ckodonnell** [comment]
Sorry, I had some issues merging and they all submitted! Is there any way to "squash" these together?

**Comment by mosabua** [comment]
you can rebase all these commits . For example see this

https://www.ksharifbd.com/blog/git-rebase-how-to-squash-multiple-commits-into-single-commit/#How-to-squash

and then force push

```
git push -f
```

**Review by mosabua — APPROVED** [review]
Excellent update. Good to go after the signed CLA is submitted.

**Comment by martint** [comment]
@cla-bot check

**Comment by martint** [comment]
Merged as 4c9c6d7c82d4bde81f44cf66dbb59cf44c958653. Thanks!

**Pull request closed without merging**